### ISSUE TITLE
tracking_id / PID prefix

### ISSUE THREAD

**Issue by sol1105** [WCRP-CORDEX/cordex-cmip6-cv#51]
Hi @larsbuntemeyer and @gnikulin ,

as `tracking_id`/PID prefix you set `"hdl:21.14100/.*"` which is the CMIP6 prefix. That one is already heavily used.
After discussing this with the PID service responsibles here at DKRZ, we would kindly ask you to use the CORDEX prefix (`"hdl:21.14103/.*"`) instead.

Thank you and kind regards,
Martin

**Comment by larsbuntemeyer** [comment]
Absolutely, thanks @sol1105 for providing the PID! This is also a relict caused by our shameless attempt to adapt the original CMIP6 CV for CORDEX!

**Comment by jesusff** [comment]
Hi Katharina (@soay),

In the CORDEX-CMIP6 archiving specifications, we have the `tracking_id` prefix `hdl:21.14103/.*`, but we recently noticed that this prefix was already in use for some older CORDEX(-CMIP5) data (e.g. [here](https://handle-esgf.dkrz.de/lp/21.14103/5074939b-caa5-45a2-8db7-40546a6d0868)). We wonder if this prefix was allocated for CORDEX in general (i.e. it would apply to any CORDEX data regardless of the DRS or ESGF project) or specifically for what was then called just CORDEX (CMIP5-driven) and we should allocate a new one for CORDEX-CMIP6. We are also planning for CORDEX-CMIP7 and this would mean still an additional handle prefix.

Thanks!

**Comment by soay** [comment]
Since there are only a very small number of Handles using the 21.14103 prefix we decided to re-use this for CORDEX-CMIP6 as well. Depending of the number of handles we are expecting for CORDEX-CMIP6 we could decide weather we can re-use it for CORDEX-CMIP7 or need to set up a new one for this.

**Comment by jesusff** [comment]
Great. Thanks!
